core/src/main/scala/org/apache/spark/rdd/PipedRDD.scala

```diff
@@ -230,7 +230,7 @@ private[spark] class PipedRDD[T: ClassTag](
   }
 }
 
-private object PipedRDD {
+object PipedRDD {
```

Comment:
This is a `core` module. We should be careful.
Maybe, `private[spark]` is enough for your PR, @viirya ?

Reply:
okay, thanks for pointing it.